test/testConfig.json

```diff
@@ -1,12 +1,14 @@
 {
     "token": {
-        "user": "wjriehl",
-        "file": "test/wjriehl.tok"
+        "user": "kbaseutest",
```

Comment:
should not have even been checked in... that was an accident.
incidentally, it would be nice if we could rearrange things so there are no straggling, incidental file changes that should not be checked in. some files like the configs are in the codebase, so cannot be gitignored, but are modified during make install or test setup, so it is pretty easy at the end of along day to accidentally include them in a commit...

Reply:
I'm going to try to get this ready this week for the upcoming release, gotta take care of the conflicts that have crept in.